web/locales/en/messages.ftl

```diff
@@ -31,38 +31,43 @@ wall-of-text-more-desktop =
 show-wall-of-text = Read More
 help-us-title = Help us validate sentences!
 help-us-explain = Press play, listen & tell us: did they accurately speak the sentence below?
+
 ## ProjectStatus
-status-title = Overall project status: see how far we've come!
+status-title = Overall project status: see how far we’ve come!
 status-contribute = Contribute Your Voice
-status-loading = Loading...
-status-hours = { $hours } validated hours so far!
+status-loading = Loading…
+status-hours =
```

Comment:
@pike 
Do you know if this require [code changes](https://github.com/mozilla/voice-web/blob/master/web/src/components/pages/home/project-status.tsx#L70)?

Reply:
nope, yay, fluent